frdrpc/faraday.proto

```diff
@@ -320,6 +320,15 @@ enum Granularity {
     DAY = 8;
 }
 
+/*
+FiatBackend is the API endpoint to be used for any fiat related queries.
+*/
+enum FiatBackend {
+    UNKNOWN_FIATBACKEND = 0;
+    COINCAP = 1;
```

Comment:
nit: comments here would be nice, perhaps just list the url that we get this info for? When we generate docs from our protos, the comments on these fields are what end users see in our API docs, so it's nice to be as verbose as possible here. 

Because we don't have docs besides API 🙈

Reply:
Great docs!